walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,69 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredAndOptionalIgnored() {
+    // Have each person have different events and an optional attendee who has a meeting the
+    // whole day. Since he doesn't have any available spots, he will be ignored and 3 ranges
+    // will be returned.
+    //
+    // Events  :       |--A--|     |--B--|
+    // Optional: |------------C----------------|
```

Comment:
I don't think you have any test which considers what would happen if optional attendees:
- have partial availability: there is no slot that works for an optional attendee, but they could attend half of the required duration for some slots. Will those slots be preferred in any way?
- if you have 2 optional attendees and 3 possible slots: 1 only works for required attendees, 1 works for first optional ateendee, 1 works for second optional attendee. Which ones would the function return?